packages/wallet/backend/src/rates/service.ts

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios'
+
+export type RatesResponse = {
+  base: string
+  rates: Record<string, number>
+}
+
+export interface IRatesService {
+  getRates: (userId: string) => Promise<RatesResponse>
```

Comment:
userId?

Reply:
@beniaminmunteanu  looks like it should be base, not userId